artilleryTests/rosetta-load-test.yml

```diff
@@ -1,5 +1,16 @@
 config:
   target: http://127.0.0.1:8082
+  payload:
+    path: 'data.csv'
+    fields:
+      - 'address'
+      - 'block'
```

Comment:
Now we have the 0th, 50th, 29th, and 22nd blocks to be checked. Do they have any transactions?

Reply:
Good question! The 50th block has, the others do not.
I'll change it.